arquillian-junit5/src/test/java/org/wildfly/guide/testing/client/AddTaskResourceTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.guide.testing.client;
+
+import java.net.URI;
+
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.ClientBuilder;
+import jakarta.ws.rs.client.Entity;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.core.UriBuilder;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit5.container.annotation.ArquillianTest;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.wildfly.guide.testing.model.Task;
+
+/**
+ * Tests that a task was added to the database.
+ *
+ * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
+ */
+@ArquillianTest
+@RunAsClient
+public class AddTaskResourceTest {
+
+    @ArquillianResource
+    private URI uri;
+
+    @Deployment
+    public static WebArchive createDeployment() {
+        return ShrinkWrap.create(WebArchive.class)
+                .addPackages(true, "org.wildfly.guide.testing")
+                .addAsResource("META-INF/persistence.xml")
+                .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml");
+    }
+
+    @Test
+    public void addTask() {
```

Comment:
This test is practically the same as CrudTaskResourceTests#addTest, is that intentional? Not sure what's the point of having it in a separate class, so maybe this class can be removed in favor of CrudTaskResourceTests

Reply:
Good catch. I'll delete this. I think I added it as the first test to make sure everything was working and forgot to delete it.